src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -83,14 +104,32 @@ public JedisCluster(HostAndPort node, final JedisClientConfig clientConfig, int
     this(Collections.singleton(node), clientConfig, maxAttempts, poolConfig);
   }
 
+  /**
+   * Creates a Jedis-Cluster with multiple Nodes/Instances.
```

Comment:
Please don't use `Jedis-Cluster`. You can use `JedisCluster` or `Cluster` or `cluster`.